src/index.html

```diff
@@ -10,7 +12,23 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form action="#"  method="get" data-qa="big">
```

Comment:
5 point in the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)